rakefile.rb

```diff
@@ -44,22 +44,19 @@
 - [ ] check build
 -  edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
     - [ ] complete release notes, mentioning non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
-    - [ ] attach nupkg
+    - [ ] attach nupkg(s) - main package and/or analyzer, whichever have new content
     - [ ] publish the release
 - [ ] push NuGet package
 - [ ] de-list pre-release or superseded buggy NuGet packages if present
 - [ ] update website with contributors list (if in place)
 - [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
 - [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
 - [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
-- [ ] use `rake set_version[new_version]` to
-    - create a new branch
-    - change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
-    - push to origin
-    - create PR to upstream master
+- [ ] use `rake set_version[new_version]` to create a pull request that changes the version in 
+       CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_), and once that PR has been merged

```

Comment:
That's a fair point. The real thing we need is to call `rake create_milestone` from a working copy that has the desired version number in it. So merging is not necessary. I will amend. Thanks.